pom.xml

```diff
@@ -123,7 +123,7 @@
                     <artifactId>mockito-all</artifactId>
                 </exclusion>
             </exclusions>
-            <version>3.1.4-SNAPSHOT</version>
+            <version>3.1.3</version>
         </dependency>
```

Comment:
Is this change intentional?

Reply:
@benj58xu Yes for now, For api we need to give release versions, I am working on release of hygieia-core once that is done this version will get changed.

Reply:
@daniel14y70huang Please set the version property of `${com.capitalone.dashboard.core.version}` to 3.1.5